.github/workflows/pyscal.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Lint with black
         if: ${{ always() }}
         run: |
-          black --check --force-exclude="pyscal/version.py" pyscal/*py tests/test_*py setup.py docs/conf.py
+          black --check --force-exclude="pyscal/version.py" pyscal/*py tests/test_*py docs/conf.py
```

Comment:
I'm skeptical that globbing only py files is needed here, but not really relevant to the purpose of this pr :)

Reply:
that's true. I have removed the globbing part.